tests/test_parser.rs

```diff
@@ -283,3 +283,20 @@ fn test_block_comments() {
     assert_to("#/**/a", "#a");
     assert_to("/*(*/", "");
 }
+
+#[test]
+fn test_source_comments() {
+    use motoko::ast::{Dec, Exp, Loc, Source};
+    use motoko::check::parse;
+    let ast = parse("//\n/*a*/a").unwrap();
+    println!("{:?}", ast);
+    let expected = Loc(
+        Box::new(Dec::Exp(Exp::Var("a".to_string()))),
+        Source::Known {
+            line: 2,
+            col: 6,
+            span: 8..9,
```

Comment:
I see what you mean about the `..` syntax.  I have not used it myself before.  Nice that it looks the same here as in our formatter.